Neos.Neos/Classes/FrontendRouting/NodeAddress.php

```diff
@@ -42,8 +32,9 @@
     /**
      * @internal use NodeAddressFactory, if you want to create a NodeAddress
      */
+    /** @phpstan-ignore-next-line its all just temporary */
```

Comment:
Could you elaborate the comment a bit? Why do we need it?

Reply:
i dindt want to remove the param yet as it would break the neos ui now (we construct it over there) this is just intermediate